.github/ISSUE_TEMPLATE.md

```diff
@@ -1,33 +1,3 @@
 <!--
-    Anything in <!-- and -> are comments and will not appear in the submitted issue.
-
-    Note:
-        This template is just a suggestion for the layout for your report.
-        As a general rule this template should be followed, but feel free to modify it according to your situation.
-        However it is important that you provide as much detail as you can possibly provide so that we can address the
-        issue with minimal fuss.
+Before reporting a bug or requesting a feature, please make sure you have as much information as possible. Check our Contributing guidelines for more on what we're looking for.
```

Comment:
You really dont really need to mention the Contributing guidelines part, as it is shown automatically by github